core/src/main/scripts/importer/validateData.py

```diff
@@ -1463,15 +1463,15 @@ def checkEndPosition(self, value):
 
     def checkDriver(self, value):
         """Validate the values in the cbp_driver column."""
-        if value not in ('Putative_Passenger', 'Putative_Driver', 'NA', 'Unknown'):
-            self.extra = 'Only "Putative_Passenger", "Putative_Driver", "NA" and "Unknown" are allowed.'
+        if value not in ('Putative_Passenger', 'Putative_Driver', 'NA', 'Unknown', ''):
+            self.extra = 'Only "Putative_Passenger", "Putative_Driver", "NA", "Unknown" and "" (empty) are allowed.'
             self.extra_exists = True
             return False
         return True
     
     def checkDriverTiers(self, value):
         """Report the tiers in the cbp_driver_tiers column (skipping the empty values)."""
-        if value is not "":
+        if value not in ('NA', ''):
```

Comment:
I think we are keeping track of such lists in variables like `NULL_AA_CHANGE_VALUES`, `NULL_VALUES`